framework/src/org/checkerframework/common/value/qual/IntRange.java

```diff
@@ -11,14 +11,17 @@
  * the given range. The bounds are inclusive; for example, {@code @IntRange(from=6, to=9)}
  * represents the four values 6, 7, 8, and 9.
  *
+ * <p>If only one of the {@code to} and {@code from} fields is set, then the other will default to
+ * the max/min value of the underlying type of the variable that is annotated.
+ *
  * @checker_framework.manual #constant-value-checker Constant Value Checker
  */
 @SubtypeOf(UnknownVal.class)
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE_PARAMETER, ElementType.TYPE_USE})
 public @interface IntRange {
     /** Smallest value in the range, inclusive */
-    long from() default Long.MIN_VALUE;
```

Comment:
Javac will complain if an annotation field doesn't have a default and a value isn't specified in an annotation.  In other words, Java is going to complain about `@IntRange(from=0)`.